src/app/services/project_service.js

```diff
@@ -148,7 +148,6 @@ angular
             
             // Add metrics back into nodes to make site logic work
             _.each(service.files.manifest.metrics, function(node) {
-                node.label = node.name;
```

Comment:
Don't update it yet.  We're having a conversation literally right now about this.  Will get back to you in 5!

Reply:
hey @GtheSheep - thanks _very_ much for digging into this! @callum-mcdata and I did something similar in https://github.com/dbt-labs/dbt-docs/pull/285 -- I wish i had saw this PR first!!

We debated internally about showing the metric label instead of the metric name in the DAG viz (and in other parts of the UI). My thinking is that it's preferable to see the label (say: `Average Revenue Per Customer`) instead of a less human-readable name (eg. `arpc`). What do you think makes more sense / you'd prefer to see?

Reply:
hey @drewbanin - yeah I think it most cases it's probably preferable to see the label, even if that's not quite consistent with some of the other objects, but if it's just a styling choice with no other impacts it's probably nicer for general users, the only slight downside I can see is the lineage graph then not matching the selector at the bottom, but that seems minimal 🤷‍♂️

Reply:
happy to close this in favour of #285 😸